app/javascript/components/recordings/Recordings.jsx

```diff
@@ -1,64 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
-  Table, Card, Row, Button,
+  Card, Button, Stack, Table,
 } from 'react-bootstrap';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faVideo, faTrashCan } from '@fortawesome/free-solid-svg-icons';
-import Modal from '../shared/Modal';
-import Spinner from '../shared/stylings/Spinner';
 import useRecordings from '../../hooks/queries/recordings/useRecordings';
 import useRecordingsReSync from '../../hooks/queries/recordings/useRecordingsReSync';
-import DeleteRecordingForm from '../forms/DeleteRecordingForm';
+import SearchBarQuery from '../shared/SearchBarQuery';
+import RecordingsList from './RecordingsList';
 
 export default function Recordings() {
-  const { isLoading, data: recordings } = useRecordings();
+  const [input, setInput] = useState('');
+  const { data: recordings } = useRecordings(input);
   const { refetch: handleRecordingReSync } = useRecordingsReSync();
 
-  if (isLoading) return <Spinner />;
```

Comment:
Why are you dropping the `isLoading`?

Reply:
Right now, the query is fired onChange and not onSubmit.
Not sure this is a good design as this will trigger a isLoading on every character.
The Spinner is really annoying.
Maybe we could solve this with placeholders.
We could talk about this in more details.

Reply:
Gotcha